kg_rag/utility.py

```diff
@@ -392,5 +392,6 @@ def interactive(question, vectorstore, node_context_df, embedding_function_for_c
         output = llm_chain.run(context=node_context_extracted, question=question)
     elif "gpt" in llm_type:
         enriched_prompt = "Context: "+ node_context_extracted + "\n" + "Question: " + question
-        output = get_GPT_response(enriched_prompt, system_prompt, llm_type, llm_type, temperature=config_data["LLM_TEMPERATURE"])
+        chat_model_id, chat_deployment_id = get_gpt35()
```

Comment:
Noticed that `chat_deployment_id` was same as `chat_model_id` if not using this step when using `GPT_API_TYPE : 'open_ai'`.

Reply:
if api type is open_ai, I think chat_deployment_id is None. Please see my response given [below](https://github.com/BaranziniLab/KG_RAG/pull/34/files#r1669915114).